docs/UserGuide.md

```diff
@@ -90,6 +90,23 @@ Shows a list of all persons in the address book.
 
 Format: `list`
 
+### Filter students by tags: `filter`
+
+Filters students who have a given tag.
+
+Format: `filter t/TAG`
+
+* The tags that can be used are: `n/STUDENT_NAME` and `ev/EVENT_NAME`
```

Comment:
Instead of ev/EVENT_NAME, maybe use e/EVENT_NAME. Cause I used e/ to format my create command, so I think we should be consistent.